### PR TITLE
test: update the existing tag image case

### PR DESCRIPTION
**Pull request by fuweid**
Signed-off-by: Wei Fu <fuweid89@gmail.com>

### Ⅰ. Describe what this PR did

The TestImageTagFailToOverrideExistingTag tests the case that it will
fail to use existing tag name to override existing tag.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

update existing case

### Ⅳ. Describe how to verify it

wait for CI

### CODE REVIEW

File: test/api_image_tag_test.go [hunk 1]
```diff
@@ -73,14 +73,29 @@ func (suite *APIImageTagSuite) TestImageTagFailToOverrideExistingPrimaryReferenc
...
-	c.Assert(err, check.IsNil)
-	CheckRespStatus(c, resp, 400)
+	// create valid tag
+	{
+		command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
+
+		resp, err := suite.sendRequest(busyboxImage, repo, tag)
+		c.Assert(err, check.IsNil)
+		CheckRespStatus(c, resp, 201)
+		defer DelImageForceOk(c, tagRef)
```
Comment (rudyfly) [inline test/api_image_tag_test.go]: why delete tagRef image？

Comment (fuweid) [reply test/api_image_tag_test.go]: the `tagRef` image is created by the case. If we don't delete it, re-run this case will fail. It just is used to clean up.

Comment (rudyfly) [reply test/api_image_tag_test.go]: But the `busyboxImage` also will be remain, dont need to delele it？

Comment (fuweid) [reply test/api_image_tag_test.go]: `busyboxImage` is created for the case suite and basically, all suites are related to this image. Therefore, we don't need to delete it.

**Comment by rudyfly** [comment]
LGTM

**Pull request merged**